Self Host Drawing Links

clayfanatic opened the pull request:
Moved the two drawing pdfs to be self hosted. Replaced the links to those in the content/docs folder.

clayfanatic committed:
Self Host Drawing Links

Moved the two drawing pdfs to be self hosted. Replaced the links to those in the content/docs folder.

moorepants commented:
The only thing about this, is that we'd need to get permission from the authors of the documents to use them. Can you find where they moved them? Or did they just delete their websites?

clayfanatic commented:
Dimensioning and Tolerancing moved: 
https://engineering.tcnj.edu/wp-content/uploads/sites/194/2012/02/dimensioning_and_tolerancing.pdf

The Creating Design Packages is not rehosted, searching through Jim Glancey's new website doesn't turn anything up.
http://udel.edu/~jglancey/Teaching.html

clayfanatic committed:
Changed Links

I have changed the first pdf link to the new website at engineering.tcnj.edu.

The second pdf is MIA on the new website; so I'm using the archive.org link you sent.

moorepants commented:
Ok, this works. Thanks.
